Dockerfile

```diff
@@ -1,14 +1,8 @@
 FROM google/cloud-sdk:170.0.1-slim
 
-RUN pip install virtualenv
-RUN virtualenv -q ~/.firecloud-tools/venv
-RUN /bin/bash -c "source ~/.firecloud-tools/venv/bin/activate"
+COPY install.sh /
+RUN /bin/bash -c "/install.sh"
 	
-RUN pip install --upgrade pip
-
-RUN pip install PyYAML google-cloud-datastore google-cloud google-auth-httplib2 google-api-python-client gcs-oauth2-boto-plugin \
```

Comment:
google-cloud-datastore & google-cloud have been removed from here but not in the install.sh. Can they be re-added? They are requirements for the cleanup_submission, I believe.